core/numbers/numbers.py

```diff
@@ -5,12 +5,12 @@
 mod = Module()
 ctx = Context()
 
-digits = "zero one two three four five six seven eight nine".split()
+digit_list = "zero one two three four five six seven eight nine".split()
```

Comment:
This variable name conflicts with the `digits` capture function name, so my linter was complaining; figured it was worth fixing

Reply:
Must say I'm not a big fan of captures with plural names that only return a single value. I call my alternative just `digit`